src/components/Confirm/Confirm.test.tsx

```diff
@@ -21,6 +23,15 @@ const render = (storeState: any = {}) => {
   };
 };
 
+const intersectionObserverMock = () => ({
```

Comment:
you can use built-in util `import { mockAllIsIntersecting } from 'react-intersection-observer/test-utils';`